src/pages/index.jsx

```diff
@@ -15,55 +20,65 @@ library.add(fas);
 const IndexPage = () => (
   <IntlProvider locale="en">
     <Layout>
-      <Hero
-        programTitle="Data Science Master's Degree"
-        organizationLogo={{
-          url: 'https://www.edx.org/sites/default/files/school/image/logo/gtx-logo-200x101.png',
-          alt: 'Georgia Tech Institute of Technology logo',
-        }}
-        textureImage="https://prod-discovery.edx-cdn.org/media/degree_marketing/campus_images/gt-cyber-title_bg_img_440x400.jpg"
-        coverImage="https://prod-discovery.edx-cdn.org/media/degree_marketing/campus_images/gt_cyber_campus_image_1000x400.jpg"
-      />
-      <div className="container py-5">
-        <div className="row">
-          <div className="col-xs-12 col-lg-8">
-            <div className="mb-5">
-              <h2 className="mb-4">My Courses In Progress</h2>
-              <InProgressCourseCard
-                title="Advanced Analytics Thinking"
-                endDate="2019-11-03"
-                linkToCourse="https://edx.org"
-              />
-              <InProgressCourseCard
-                title="Statistics for Analyics"
-                endDate="2019-07-23"
-                linkToCourse="https://edx.org"
-                microMastersTitle="MicroMasters&reg; Program in Analytics: Essential Tools and Methods"
-              />
-            </div>
-            <div className="mb-5">
-              <h2 className="mb-4">Upcoming Courses</h2>
-              <UpcomingCourseCard
-                title="Predictive Analytics with Spark"
-                startDate="2019-04-29"
-                linkToCourse="https://edx.org"
-              />
+      <ConnectedRouter history={history}>
+        <React.Fragment>
+          <PrivateRoute
+            path="/"
+            component=""
+            authenticatedAPIClient={apiClient}
+            redirect={process.env.BASE_URL}
+          />
+          <Hero
```

Comment:
I assume this private route is here to protect the data on the rest of the page. if thats the case, we should export the `<Hero/>` and course cards to a separate component and render it with the private routes' `component={}`.

Is that correct @adamstankiewicz ? or is the `<Hero/>` always supposed to be there?

Reply:
Refactored